tpe/plugin/src/FreeGroupFeatures.cc

```diff
@@ -36,8 +36,8 @@ Identity FreeGroupFeatures::FindFreeGroupForModel(
   auto it = this->models.find(_modelID.id);
   if (it == this->models.end() || it->second == nullptr)
     return this->GenerateInvalidId();
-  // if there are no links in this model, then the FreeGroup functions
-  // will not work properly; need to reject this case.
+  // if there are no links or nested modelsin this model, then the FreeGroup
```

Comment:
`modelsin` -> `models in`

Reply:
Done in https://github.com/ignitionrobotics/ign-physics/pull/231/commits/74b3f6d75d66eef7fb9af2caf5697e1629cb5046